prow/plugins.yaml

```diff
@@ -39,3 +39,7 @@ plugins:
   - shrug
   - wip
   - yuks
+
+  GoogleCloudPlatform/osconfig:
+  - config-updater
```

Comment:
config all lives in this oss-test-infra repo, you don't need config-updater in your repo

Reply:
Removed.